packages/main/test/samples/SplitButton.sample.html

```diff
@@ -53,4 +53,86 @@ <h3>SplitButton with Icons</h3>
+		<ui5-menu id="menuInSplitBtnDefaultAction">
+			<ui5-menu-item text="Edit" icon="add"></ui5-menu-item>
+			<ui5-menu-item text="Save" icon="save"></ui5-menu-item>
+			<ui5-menu-item text="Delete" icon="delete"></ui5-menu-item>
+		</ui5-menu>
+	</div>
+	<pre class="prettyprint lang-html"><xmp>
+<ui5-split-button id="splitBtnWithMenuDefaultAction" design="Default">Open Menu</ui5-split-button>
+<ui5-menu id="menuInSplitBtnDefaultAction">
+	<ui5-menu-item text="Edit" icon="add"></ui5-menu-item>
+	<ui5-menu-item text="Save" icon="save"></ui5-menu-item>
+	<ui5-menu-item text="Delete" icon="delete"></ui5-menu-item>
+</ui5-menu>
+<script>
+	const splitBtnWithMenuDefaultAction = document.getElementById("splitBtnWithMenuDefaultAction");
+	const menuInSplitBtn = document.getElementById("menuInSplitBtnDefaultAction");
+
+	splitBtnWithMenuDefaultAction.addEventListener("ui5-arrow-click", function () {
+		menuInSplitBtn.open ? menuInSplitBtn.close() : menuInSplitBtn.showAt(splitBtnWithMenuDefaultAction);
+	});
+</script>
+	</xmp></pre>
+
+	<script>
+		const splitBtnWithMenuDefaultAction = document.getElementById("splitBtnWithMenuDefaultAction");
+		const menuInSplitBtn = document.getElementById("menuInSplitBtnDefaultAction");
+
+		splitBtnWithMenuDefaultAction.addEventListener("ui5-arrow-click", function () {
+			menuInSplitBtn.open ? menuInSplitBtn.close() : menuInSplitBtn.showAt(splitBtnWithMenuDefaultAction);
+		});
+	</script>
+</section>
+<section>
+	<h3>SplitButton with Menu and associated last action</h3>
+	<div class="snippet">
+		<ui5-split-button id="splitBtnWithMenuWithAssociatedLastAction" class="samples-margin" design="Default">Open Menu</ui5-split-button>
+		<ui5-menu id="menuInSplitBtnAssociatedLastAction">
+			<ui5-menu-item text="Edit" icon="add"></ui5-menu-item>
+			<ui5-menu-item text="Save" icon="save"></ui5-menu-item>
+			<ui5-menu-item text="Delete" icon="delete"></ui5-menu-item>
+		</ui5-menu>
+	</div>
+
+	<pre class="prettyprint lang-html"><xmp>
+<ui5-split-button id="splitBtnWithMenuWithAssociatedLastAction" design="Default">Open Menu</ui5-split-button>
+<ui5-menu id="menuInSplitBtnAssociatedLastAction">
+	<ui5-menu-item text="Edit" icon="add"></ui5-menu-item>
+	<ui5-menu-item text="Save" icon="save"></ui5-menu-item>
+	<ui5-menu-item text="Delete" icon="delete"></ui5-menu-item>
+</ui5-menu>
+
+<script>
+	const splitBtnLastAction = document.getElementById("splitBtnWithMenuWithAssociatedLastAction")
+	const menuInSplitBtnLastAction = document.getElementById("menuInSplitBtnAssociatedLastAction")
+
+	splitBtnLastAction.addEventListener("ui5-arrow-click", function () {
+		menuInSplitBtnLastAction.open ? menuInSplitBtnLastAction.close() : menuInSplitBtnLastAction.showAt(splitBtnLastAction);
+	});
+
+	menuInSplitBtnLastAction.addEventListener("ui5-item-click", function(event) {
+		splitBtnLastAction.innerText = event.detail.text;
+	});
+</script>
+	</xmp></pre>
+
+	<script>
```

Comment:
move that before `pre` tag otherwise someone might not associate it with the current sample